config/config_test.go

```diff
@@ -0,0 +1,13 @@
+package config
+
+import (
+	"testing"
+)
+
+func TestGetConfiguration(t *testing.T) {
+	configuration := *GetConfiguration()
+
+	if configuration.Port == "" && configuration.MongoUrl == "" && configuration.DbName == "" && configuration.Collection == "" && configuration.PokeApiUrl == "" {
```

Comment:
For testing config loading, you would like to have a test env file(probably in a test folder inside your package to separate things), with test values, you could have a file with correct mocked values, another file with incorrect values to test how your function would handle those scenarios.

This would imply having a way to pass to your function the file path you're trying to get

Reply:
:ok_hand: